src/functions/template-tags/organizer.php

```diff
@@ -243,6 +243,11 @@ function tribe_get_organizer_link( $postId = null, $full_link = true, $echo = fa
 				return apply_filters( 'tribe_get_organizer_link', $link, $postId, $full_link, $url );
 			}
 		}
+
+		if ( ! $full_link ) {
```

Comment:
1. $echo is deprecated 
2. If PRO isn't active and you send $full_link as false, it'll always return the organizer name, which is not useful or expected in that scenario
3. I am able to reproduce the problem